do

```diff
@@ -15,211 +15,450 @@ from epimodel.gleam import Batch, batch
 log = logging.getLogger("do")
 
 
-def import_countermeasures(args):
-    log.info(f"Importing countermeasures from {args.SRC} into {args.DEST} ...")
-    cms = epimodel.imports.import_countermeasures_csv(args.rds, args.SRC)
-    cms.to_csv(args.DEST)
-    log.info(
-        f"Saved countermeasures to {args.DEST}, {len(cms.columns)} features, "
-        f"last day is {cms.index.get_level_values(1).max()}"
+# Global arguments
+
+
+@click.group()
+@click.option("-d", "--debug", is_flag=True, help="Enable debugging logs.")
+@click.option(
+    "-C",
+    "--config",
+    type=click.Path(exists=True),
+    default="config.yaml",
+    envvar="EPI_CONFIG",
+    help="Path to config file (default config.yaml; alternatively set EPI_CONFIG environment variable).",
+)
+@click.pass_context
+def cli(ctx, debug, config):
+    """
+    Epimodel pipeline runner.
+
+    See https://github.com/epidemics/epimodel for more details.
+
+    1. Update Johns Hopkins data:
+    
+    ./do action update-johns-hopkins (not needed if you got fresh data from the repo)
+
+    2. Generate batch file from estimates and basic Gleam XML definition. 
+    
+    ./do action generate-gleam-batch -D 2020-04-15 -c JK default.xml estimates-2020-04-15.csv
+
+    The batch file now contains all the scenario definitions and initial populations. Note the estimate input specification may change.
+
+    3. Export Gleam simulation XML files in Gleamviz (not while gleamviz is running!). 
+
+    ./do action export-gleam-batch out/batch-2020-04-16T03:54:52.910001+00:00.hdf5
+
+    4. Start gleamviz. You should see the new simulations loaded. Run all of them and "Retrieve results" (do not export manually). Exit gleamviz.
+
+    5. Import the gleamviz results into the HDF batch file (Gleamviz must be stopped before that). After this succeeds, you may delete the simulations from gleamviz.
+
+    ./do action import-gleam-batch out/batch-2020-04-16T03:54:52.910001+00:00.hdf5 
+
+    6. Generate web export (additional data are fetched from config.yml)
+
+    ./do action web-export out/batch-2020-04-16T03:54:52.910001+00:00.hdf5 data/sources/estimates-JK-2020-04-15.csv
+
+    7. Export the generated folder to web! Optionally, set a channel for testing first. 
+
+    ./do action web-upload out/export-2020-04-03T02:03:28.991629+00:00 -c ttest28
+    """
+    ctx.ensure_object(dict)
+    ctx.obj["DEBUG"] = debug
+
+    logging.basicConfig(level=logging.INFO)
+    if debug:
+        logging.root.setLevel(logging.DEBUG)
```

Comment:
I would personally write it this way but it's a matter of taste
```suggestion
    logging.basicConfig(level=logging.DEBUG if debug else logging.INFO)
```

Reply:
Made the change :)